test_suite/cloud/test_aws.py

```diff
@@ -211,6 +212,44 @@ def test_hostkey_permissions(self, host):
                 assert host.file(f'/etc/ssh/{file}').mode >= 0o640, \
                     'ssh files permissions are not set correctly'
 
+    def test_aws_instance_identity(self, host, instance_data):
```

Comment:
Just curious, why don´t we include ami-val docstrings? I think the save some time when trying to understand the test

Reply:
Good point. It makes even more sense now that we have the description of the HTML report directly obtained from doctstrings. Let's do it for all the migrated tests! I will review the currently migrated ones and update them accordingly (in a new commit, to ease the review).

Reply:
🚀🚀🚀